### ISSUE TITLE
Add to GNU ELPA ?

### ISSUE THREAD

**Issue by phil-s** [Wilfred/helpful#1]
Well this package seems nice.

Seeing as it is for Emacs Lisp reference, are you planning to put it into the GNU ELPA archive?

**Comment by Wilfred** [comment]
I'd like to, but it's currently blocked on https://github.com/Wilfred/elisp-refs/issues/1 which has a lot of dependencies outside of GNU ELPA.

It's not a priority, but I would like to do this.
